test/pytorch.jl

```diff
@@ -0,0 +1,195 @@
+using CounterfactualExplanations
+using CounterfactualExplanations.Evaluation
+using CounterfactualExplanations.Generators
+using CounterfactualExplanations.Data
+using CounterfactualExplanations.Models
+using CounterfactualExplanations.DataPreprocessing
+using DataFrames
+using Flux
+using LinearAlgebra
+using MLUtils
+using Random
+using PythonCall
```

Comment:
Actually, all of these should be in the "base" file `runtests.jl`. Can you please open an issue for this?

Reply:
an Issue, because other files also don't follow this format

Reply:
no, not required